wiki/Performance_Points/Performance_Point_Systems/en.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
I don't think we need this; this is literally just a smaller-ish version of the actual peformance points article, which has all of this info and more.